docs/changelog.rst

```diff
@@ -5,6 +5,7 @@ Changelog
 
 `Unreleased`_
 -------------
+- ``operationId`` argument can  get operations accurately by operationId (Only with OpenAPI 3.0+)
```

Comment:
Please, remove double space after "can" and add an empty line before this one. Also, I think that if we follow the general naming convention in the repo, then the argument name should be `operation_id`

Reply:
Yes, I modified it, and it conflicts with the latest file. I'll continue later